tests/test_api_consistency.py

```diff
@@ -214,4 +218,5 @@ def test_api_update_index():
         False,
         None,
         None,
+        inspect._empty,
```

Comment:
It looks like we aren't too far off from the `getfullargspec` version, `parameters` appears to include the positional and `**kwargs` parameters. `getfullargspec()` doesn't return anything for a `@wraps` function...